### ISSUE TITLE
Plugin not working properly if WordPress doesn't have write permissions inside plugin and uploads folder

### ISSUE THREAD

**Issue by zauan** [zionbuilder/zionbuilder#281]
More information can be found here https://wordpress.org/support/topic/fatal-error-when-activating-the-plugin-on-a-local-machine/#post-13797599

**Comment by swinggraphics** [comment]
_WordPress_ (not just this plugin) doesn't work properly if it can't write to those directories, right? It means you can't add or update plugins or upload media.

**Comment by zauan** [comment]
Indeed, however, we can display a warning message in this cases so the end user knows why the plugin doesn't work

**Comment by swinggraphics** [comment]
👍 Gotcha
